imports/node-app/core/createApolloServer.js

```diff
@@ -35,9 +35,19 @@ const resolverValidationOptions = {
 export default function createApolloServer(options = {}) {
   // the Meteor GraphQL server is an Express server
   const expressServer = express();
-
-  const { addCallMeteorMethod, context: contextFromOptions, resolvers, typeDefs } = options;
+  let typeDefs = options.typeDefs || [];
```

Comment:
Since it can now be schemas or typeDefs, maybe rename the `typeDefs` option to `schemas`? That term seems more generic.

Reply:
OK are these options part of our external API surface? If I just make sure the reaction codebase is updated, all good?